src/provider/config-provider.js

```diff
@@ -1,20 +1,22 @@
 import { writable } from "svelte/store";
+import allLocale from "./locale";
```

Comment:
I would also import just the `./local/default` so we don't package in all the locals